ceno_zkvm/src/scheme/cpu/mod.rs

```diff
@@ -55,6 +55,7 @@ pub type TowerRelationOutput<E> = (
 
 // accumulate N=2^n EC points into one EC point using affine coordinates
 // in one layer which borrows ideas from the [Quark paper](https://eprint.iacr.org/2020/1275.pdf)
+#[derive(Default)]
```

Comment:
we can remove 
- `&self` from create_ecc_proof, 
- new() as it's do nothing currently

so the clippy will not complain about this?